packages/react-component-library/src/components/TopLevelNavigation/Masthead/Masthead.stories.tsx

```diff
@@ -118,7 +118,7 @@ export const CustomLogo: ComponentStory<typeof Masthead> = (props) => (
 
 CustomLogo.storyName = 'Custom logo'
 CustomLogo.args = {
-  onSearch: null as undefined,
+  onSearch: null,
```

Comment:
I may have commented on this on a previous PR – the problem is that if `onSearch` is set, the component shows the search button, and Storybook is injecting a value for `onSearch` if it's undefined.

Probably it'd be better if the component had a separate `hasSearch` prop but I've just allowed `null` for now.